apps/ledger-live-desktop/src/renderer/icons/Cic.jsx

```diff
@@ -0,0 +1,24 @@
+// @flow
+
+import React from "react";
+
+const Cic = ({ size, color = "currentColor" }: { size: number, color?: string }) => (
+  <svg width={size} height={size}>
+    <g clipPath="url(#clip0_2553_24519)">
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M0 20.4387V28C8.37632 28 15.1667 21.732 15.1667 14C15.1667 6.26801 8.37632 0 0 0V7.56125C3.86599 7.56125 7 10.444 7 14C7 17.556 3.86599 20.4387 0 20.4387Z"
+        fill="#3A57CA"
+      />
+      <path
+        fillRule="evenodd"
+        clipRule="evenodd"
+        d="M27.1667 8.46154V0C18.7903 0 12 6.26801 12 14C12 21.732 18.7903 28 27.1667 28V19.5385C23.853 19.5385 21.1667 17.0588 21.1667 14C21.1667 10.9412 23.853 8.46154 27.1667 8.46154Z"
+        fill="#F57382"
+      />
+    </g>
+  </svg>
+);
+
+export default Cic;
```

Comment:
This looks like a duplication of `apps/ledger-live-desktop/src/renderer/icons/providers/Cic.jsx`.

Furthermore, it does not seem to be used anywhere.

What's the use of this icon? And what's the difference with the one in `/providers`?

Reply:
Agree but I want maintain the same structure that we have now for any other provider. 
On the other hand, there is another ticket to use core icons https://ledgerhq.atlassian.net/browse/LIVE-4500 (when we move to core ui component we will remove these 2 icons)

Reply:
> in this pr I want maintain the same structure that we have now for any other provider

I get that. I think there is a confusion here.
I guess when you say "same structure" you are referring to the 2 Changelly icons:
- `apps/ledger-live-desktop/src/renderer/icons/Changelly.jsx`
- `apps/ledger-live-desktop/src/renderer/icons/providers/Changelly.jsx`

But, the `apps/ledger-live-desktop/src/renderer/icons/Changelly.jsx` one is unused (dead code).

Suggestion: instead of creating a duplicated Cic icon (that is used nowhere) in `renderer/icons/` and `renderer/icons/providers` to mimic what is existing with Changelly, we should remove this deprecated file `apps/ledger-live-desktop/src/renderer/icons/Changelly.jsx` and just create a Cic icon in the `renderer/icons/providers/` folder.

At the end of the day, we go from (currently on `develop`):
```
renderer/icons/Changelly.jsx
renderer/icons/providers/Changelly.jsx
```

To (in this PR):
```
renderer/icons/providers/Changelly.jsx
renderer/icons/providers/Cic.jsx
```

Reply:
In the next pr we are going to remove ANY provider icon from lld or llm  "https://ledgerhq.atlassian.net/browse/LIVE-4500", using ui core dependency. 
In this pr I do not think we must change/delete any file, base in "timeline" and "test effort".